src/fmu/dataio/_model/schema.py

```diff
@@ -102,7 +103,7 @@ def _validate_input(cls, values: dict) -> dict:
 class JsonSchemaMetadata(BaseModel, populate_by_name=True):
     schema_: AnyHttpUrl = Field(
         alias="$schema",
-        default=TypeAdapter(AnyHttpUrl).validate_python(FmuResultsSchema.PROD_URL),
+        default=TypeAdapter(AnyHttpUrl).validate_python(FmuResultsSchema.url()),
```

Comment:
I don't really like this TypeAdapter.. looks messy. I think we can do this instead 🙂
```suggestion
        default=AnyHttpUrl(FmuResultsSchema.url()),
```

Reply:
Same! Updated